index.html

```diff
@@ -0,0 +1,371 @@
+                    
+                </div>  
+            </div>
+            <div class="right">
+                <div class="image-box">
+                    <img src="images/home.png" alt="">
+                </div>
+            </div>
+        </div>
+    </secttion>
+<!---------------end of home---------------->
+<!----------------about mr--------------------->
+    <section class="about" id="about">
+        
+        <div class="header">
+            <h1>about me</h1>
+        </div>
+        <div class="container">   
+            
+            <div class="left">
+                <div class="image-box">
+                    <img src="images/home.png" alt="">
+                </div>
+                <div class="about-text">
+                    <p>Lorem ipsum dolor sit amet, consectetur
+                         adipisicing elit. Obcaecati, in ut? 
+                         Tenetur itaque cum voluptatibus in et
+                          provident recusandae maiores corpori
+                          s culpa, quae, inventore odit, velit
+                           quidem facilis expedita sunt?
+                    </p>
+                </div>   
+            </div>
+            <div class="right">
+                <h2>My Skills</h2>
+                <div class="skills">
+                    <div class="skill-box">html</div>
+                    <div class="skill-box">css</div>
+                    <div class="skill-box">javascript</div>
+                    <div class="skill-box">php</div>
+                    <div class="skill-box">phython</div>
+                    <div class="skill-box">mysql</div>
+                    <div class="skill-box">c++</div>
+                    <div class="skill-box">photoshop</div>
+                </div>
+                <h2>language proficiency</h2>
+                <div class="language">
+                    <div class="lang-bar">
+                        <div class="lang-box" style="width: 80%;">english</div>
+                    </div>
+                    <div class="lang-bar">
+                        <div class="lang-box" style="width: 90%;">tamil</div>
+                    </div>
+                    <div class="lang-bar">
+                        <div class="lang-box" style="width: 50%;">sinhala</div>
+                    </div>
+                    
+                    
+                    
+                    
+                   
+                </div>
+                <h2>other activities</h2>
+                <div class="misc">
+                    <div class="misc-box">
+                        <div class="icon">
+
+                        </div>
+                    </div>
+
+                </div> 
+            </div>
+        </div>
+    </section>
+    <!----------about end----------->
+    <!---------experience start----------->
+    <section class="exped" id="exped">
+        <div class="header">
+            <h1>
+                education & experience
+            </h1>

```

Comment:
shall we divide both into two separate sections? it's a bit confusing. Top it says Experience and when it comes to this places it shows Education and Experience